src/components/auth/Social.js

```diff
@@ -0,0 +1,110 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router-dom';
+import FacebookLogin from 'react-facebook-login/dist/facebook-login-render-props';
+import { GoogleLogin } from 'react-google-login';
```

Comment:
Great job in making use of react social login libraries. i.e FacebookLogin and GoogleLogin.